src/mcMd/analyzers/system/ClusterHistogram.cpp

```diff
@@ -191,50 +191,17 @@ namespace McMd
          allCOMs.allocate(identifier_.nCluster());
          allRgTensors.allocate(identifier_.nCluster());
          Molecule::ConstAtomIterator atomIter;
+         Boundary* boundaryPtr = &system().boundary();
```

Comment:
The definition of boundaryPtr  will become unnecessary if you pass system().boundary() as a 
reference.